recipes/findiff/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "findiff" %}
+{% set version = "0.8.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c89eade4ad7f7adfa3871ba00235c582ef42b6dba0e2d84c78e242be70cdbe64
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip >=21.0.1
+    - pytest-runner >=5.3.0
+  run:
+    - numpy >=1.14.2
+    - python >=3.6
+    - scipy >=1.3.1
+    - sympy >=1.8
+    - pytest-runner >=5.3.0
+
+test:
```

Can you add a pip check test as well?
```suggestion
test:
      commands:
        - pip check
      requires:
        - pip
```